packages/block-library/src/navigation-menu-item/editor.scss

```diff
@@ -50,14 +50,22 @@
 }
 
 .wp-block-navigation-menu-item {
-	font-family: $editor-font;
-	font-weight: bold;
-	font-size: $text-editor-font-size;
-	background-color: var(--background-color-menu-link);
-	color: var(--color-menu-link);
+	&:not(.is-editing) {
+		font-family: $editor-font;
+		font-weight: bold;
+		font-size: $text-editor-font-size;
+		background-color: var(--background-color-menu-link);
+		color: var(--color-menu-link);
+	}
 
-	&:focus {
+	&.is-editing .components-text-control__input {
```

Comment:
Alternatively, for the input, we could set `background-color: transparent` and `color: inherit`.

Reply:
I think the difference is that one way the whole block has the background color, and the other only the input does. Not sure which is better.

Reply:
yes, in other words, with CSS vars we're applying the colors just to `NavigationMenuItem`, not to the `NavigationMenu`